demos/ssh-demo/README.md

```diff
@@ -7,6 +7,20 @@ It cannot be established by unauthorized parties, such as someone who controls t
 The container image contains the SSH host key that can be used for impersonating the host we will connect to.
 Because this container image is encrypted, and the key to decrypting this image is only provided in measurable ways (e.g. attestation or encrypted initrd), and because the pod/guest memory is protected, even someone who controls the node cannot steal this key.
 
+## Using a pre-provided container image
+
+If you would rather build the image with your own keys, skip to [Building the container image](#building-the-container-image).
+
+A demo image is provided at [docker.io/katadocker/ccv0-ssh](https://hub.docker.com/r/katadocker/ccv0-ssh).
+It is encrypted with [Attestation Agent](https://github.com/confidential-containers/attestation-agent)'s [offline file system key broker](https://github.com/confidential-containers/attestation-agent/tree/64c12fbecfe90ba974d5fe4896bf997308df298d/src/kbc_modules/offline_fs_kbc) and [`aa-offline_fs_kbc-keys.json`](./aa-offline_fs_kbc-keys.json) as its key file.
```

Comment:
Shouldnt the offline KBC URL simply be https://github.com/confidential-containers/attestation-agent/tree/main/src/kbc_modules/offline_fs_kbc ?

Reply:
I was slightly hesitant to just post a link to the main branch. We have no (checked) versioning at this time and if we make incompatible changes, this will become invalid until updated. Right now it's 1.0.0, but have we agreed on bumping that when we make incompatible changes?